src/test/java/Homework20.java

```diff
@@ -0,0 +1,40 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest {
+    @Test
+    public static void playSong() {
+        loginKoel();
+        navigateToAllSongs();
+        songDoubleClick();
+        songIsPlaying();
+    }
+    @Test
+    public static void CreatePlaylist() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void  createPlaylist()`